### PR TITLE
Need to pass if  integral have symbolic upper and lower bound.

### LINKED ISSUE #10434: Integral is not solved
1/(2*pi)*integrate(exp(1j*w*n), (w, -wc, wc))

should evaluate to

(sin(wc)*n)/(pi*n)

### PR DESCRIPTION
**Pull request by Shekharrajak**
Tried to fix : https://github.com/sympy/sympy/issues/10434
`1/(2 * pi) * integrate(exp(1j * w * n), (w, -wc, wc))`
gives answer in `exp` form ,as generally sympy gives.
When we rewrite the answer is `sin` and simplify we get the desired answer :  
`(sin(wc) * n)/(pi * n)`

### CODE REVIEW

**Comment by Shekharrajak** [comment]
ping @smichr

**Comment by Shekharrajak** [comment]
ping @jksuom .

File: sympy/functions/elementary/piecewise.py [hunk 1]
```diff
@@ -215,6 +215,8 @@ def _eval_interval(self, sym, a, b):
         mul = 1
         if (a == b) == True:
             return S.Zero
+        elif isinstance((a,b) , Symbol) and (a.atoms(Symbol) or b.atoms(Symbol)):
```
Comment (jksuom) [inline sympy/functions/elementary/piecewise.py]: I believe ``isinstance((a,b) , Symbol)`` is always ``False`` as a tuple can hardly be a ``Symbol``.

File: sympy/integrals/tests/test_integrals.py [hunk 2]
```diff
@@ -1117,5 +1117,15 @@ def test_issue_4950():
         -2.4*exp(8*x) - 12.0*exp(5*x)
 
 
+def test_issue_10434():
+    w = symbols('w' , positive = True)
```
Comment (jksuom) [inline sympy/integrals/tests/test_integrals.py]: It seems that the integral is evaluated even in current master if ``positive=True`` is assumed.

Comment (Shekharrajak) [inline sympy/integrals/tests/test_integrals.py]: Yes ! I am closing this PR. Thanks.

**Pull request closed without merging**